### DISCUSSION TITLE
How to move the "Sessions" window to the left?

### DISCUSSION THREAD

**Discussion by RuralHunter** [kingToolbox/WindTerm#881] [Q&A] [answered]
The default position of "Sessions" window is at the right. How to move it to the left? It seems I can not drag/drop it to the left.

**Comment by kingToolbox** [comment, marked as answer]
Hello, although drag-and-drop is not currently supported, it is still very simple to adjust the layout. You can refer to [UI / Dock And Pane](https://kingtoolbox.github.io/2021/06/28/ui_dock_and_pane/) to learn more.

**Comment by RuralHunter** [comment]
Good, thank you!
